x-pack/test/fleet_api_integration/apis/epm/index.js

```diff
@@ -14,7 +14,7 @@ export default function loadTests({ loadTestFile }) {
     loadTestFile(require.resolve('./file'));
     loadTestFile(require.resolve('./template'));
     loadTestFile(require.resolve('./ilm'));
-    // loadTestFile(require.resolve('./install_bundled'));
+    loadTestFile(require.resolve('./install_bundled'));
```

Comment:
This was somehow commented out and missed in a previous PR.

Reply:
I had to make some additional changes to get these tests passing. Since they were commented out incidentally, there were some gotchas to sort out with bundled package tests in CI. Namely, we needed a way to place fixture test files in the bundled package directory, which proved difficult in an integration test environment where we'd need to mutate the `build/` directory in order to properly copy over fixture files. 

The solution I landed on with some input from the ops folks was to introduce a `developer.bundledPackageLocation` option that allows us to override Fleet's default bundled package location with one `/tmp` to prevent mutation of the build artifact during CI. It was quite the runaround (the commit log for this PR is a sight to behold), but I think this is probably the cleanest solution we can have to support this very specific use case in tests.